src/ClockPanel.test.tsx

```diff
@@ -1,65 +1,101 @@
-import { ClockPanel } from "ClockPanel";
-import {render} from '@testing-library/react'
-import { FieldConfigSource, ScopedVars, LoadingState, getDefaultTimeRange } from "@grafana/data";
-import { ClockMode, ClockRefresh, ClockType, FontWeight, ZoneFormat } from "types";
-import React from "react";
+import { ClockPanel } from 'ClockPanel';
+import { act, render } from '@testing-library/react';
+import { FieldConfigSource, ScopedVars, LoadingState, getDefaultTimeRange } from '@grafana/data';
+import { ClockMode, ClockRefresh, ClockType, FontWeight, ZoneFormat } from 'types';
+import React from 'react';
 
 describe('ClockPanel', () => {
   beforeAll(() => {
     jest.useFakeTimers();
-    jest.setSystemTime(new Date('20 Aug 2020 02:12:00 GMT').getTime())
+  });
+
+  beforeEach(() => {
+    jest.setSystemTime(new Date('20 Aug 2020 02:12:00 GMT').getTime());
   });
 
   afterAll(() => {
     jest.useRealTimers();
   });
 
-  test('renders with default settings without error', () => {
-    // ARRANGE
+  test('renders with default settings without error (24H format)', () => {
     const props = getDefaultProps();
-    const { container } = render(<ClockPanel {...props} />)
-    // ACT
+    const { container } = render(<ClockPanel {...props} />);
 
-    // ASSERT
-    expect(container).toHaveTextContent('02:12:00')
+    expect(container).toHaveTextContent('02:12:00');
+    // no timezone by default
+    expect(container.querySelector('[data-testid="time-zone"]')).not.toBeInTheDocument();
```

Comment:
going the test-id route since I can't reliable get a timezone string from jest by setting environment variables or mocking methods.